src/dash/models/DashManifestModel.js

```diff
@@ -41,6 +41,12 @@ import ObjectUtils from '../../streaming/utils/ObjectUtils';
 import URLUtils from '../../streaming/utils/URLUtils';
 import FactoryMaker from '../../core/FactoryMaker';
 
+Number.isInteger = Number.isInteger || function (value) {
```

Comment:
I've moved this polyfill function in the DashManifestModel as expected by mostly people. ;-)